Rakefile

```diff
@@ -22,3 +22,90 @@ RSpec::Core::RakeTask.new(:specs)
 
 desc 'Run RuboCop on the lib/specs directory'
 RuboCop::RakeTask.new(:rubocop)
+
+desc 'Generate the docs using YARD'
+task :doc do
+  sh('yard', 'doc')
+  # Open generated doc in browser
+  sh('open', 'yard-doc/index.html')
```

Comment:
This is not the default location where the documentation is generated iinm. This is the location in `release-toolkit` because that output dir is set [via the `.yardopts` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/.yardopts)

Reply:
As I've tested it and it worked, so I didn't realize it. But it turns out Dangermattic also [has a .yardopts file](https://github.com/Automattic/dangermattic/blob/trunk/.yardopts).

Reply:
🤦 I thought I checked that but apparently I missed this file in there too 😅 👍